examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -159,6 +159,49 @@ class _SliderDemoState extends State<SliderDemo> {
                 const Text('Continuous'),
               ],
             ),
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: Slider(
+                        value: _value,
+                        min: 0.0,
+                        max: 100.0,
+                        onChanged: (double value) {
+                          setState(() {
+                            _value = value;
+                          });
+                        },
+                      ),
+                    ),
+                    Semantics(
+                      label: 'Editable numerical value',
+                      child: Container(
+                        width: 48,
+                        height: 48,
+                        child: TextField(
+                          onSubmitted: (String value) {
+                            final double myDouble = double.tryParse(value);
+                            if (myDouble != null) {
+                              setState(() {
```

Comment:
This is fine however it would be good to remind readers not to call setState() unless the value has actually changed. Something like: 
```
final double newValue = double.tryParse(value);
if (newValue != null && newValue != _value) {
  setState(() {
    _value = myDouble.clamp(0, 100);
  });
}
```